setup/productionize/caching/externalize-redis.md

```diff
@@ -3,6 +3,7 @@ layout: single
 title:  "Externalize Redis"
 sidebar:
   nav: setup
+redirect_from: /setup/scaling/externalize-redis/
```

Comment:
Do we need the redirect? This is all new content, right?

Reply:
This has already been linked to